scripts/process.rs

```diff
@@ -0,0 +1,59 @@
+use std::{
```

Comment:
blocking: I don't think we should have this file.
- Have you considered using something like [`duct`](https://docs.rs/duct/latest/duct/) over rolling your own?
- I think we're essentially wrapping `process::Command` rather than working _with_ that API. `run` and `pipe` could be free functions, and you then inherit `process::Command`